inspire/base/templates/format/record/Inspire_Default_HTML_brief.tpl

```diff
@@ -106,18 +106,19 @@
           <div class="modal-content">
             <div class="modal-header">
               <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-              <h4 class="modal-title" id="myModalLabel">Export Article</h4>
+              <h4 class="modal-title" id="myModalLabel">Cite Article</h4>
+              <div class="row"><div class="col-md-12"><p></p></div></div>
               <div class="btn-group">
-              <button type="button" class="btn btn-primary dropdown-toggle" id="btn-drop" data-toggle="dropdown" aria-expanded="false">
-                Format: <span id="format{{record['control_number']}}"></span> <span class="caret"></span>
-              </button>
-              <ul class="dropdown-menu" role="menu">
-                <li><a class="pointer bibtex" id="bibtex{{record['control_number']}}" data-recid="{{record['control_number']}}">BibTex</a></li>
-                <li><a class="pointer latex_eu" id="latex_eu{{record['control_number']}}" data-recid="{{record['control_number']}}">LaTex(EU)</a></li>
-                <li><a class="pointer latex_us" id="latex_us{{record['control_number']}}" data-recid="{{record['control_number']}}">LaTex(US)</a></li>
-              </ul>
-            </div>
-                <button type="button" class="btn  btn-primary">Download</button>
+                <button type="button" class="btn btn-primary dropdown-toggle" id="btn-drop" data-toggle="dropdown" aria-expanded="false">
+                  Format: <span id="format{{record['recid']}}"></span> <span class="caret"></span>
+                </button>
+                <ul class="dropdown-menu" role="menu">
+                  <li><a class="pointer bibtex" id="bibtex{{record['recid']}}" data-recid="{{record['recid']}}">BibTex</a></li>
+                  <li><a class="pointer latex_eu" id="latex_eu{{record['recid']}}" data-recid="{{record['recid']}}">LaTex(EU)</a></li>
+                  <li><a class="pointer latex_us" id="latex_us{{record['recid']}}" data-recid="{{record['recid']}}">LaTex(US)</a></li>
+                </ul>
+              </div>
+              <a type="button" id="download{{record['recid']}}" class="btn btn-primary">Download</a>
```

Comment:
We should use:

```python
record['control_number']
```

As I have found some problems using the field `recid`. If you check on ES, you will see that the field with the record id is called in that way.